src/resources/js/views/scripts.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since  TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events = tribe.events || {};
+tribe.events.views = tribe.events.views || {};
+
+/**
+ * Configures Views Object in the Global Tribe variable
+ *
+ * @since  TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events.views.scripts = {};
+
+/**
+ * Initializes in a Strict env the code that manages the Event Views
+ *
+ * @since  TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} _   Underscore.js
+ * @param  {PlainObject} obj tribe.events.views.scripts
+ *
+ * @return {void}
+ */
+( function( $, _, obj ) {
+	'use strict';
+	var $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		tooltip: '.tribe-events-tooltip',
+		tooltipContent: '.tribe-events-tooltip__content',
+	};
+
+	/**
+	 * Initialize accessible tooltips via tooltipster
+	 *
+	 * @since TBD
+	 *
+	 */
+	obj.initTooltips = function() {
```

Comment:
Should we use the same structure for the method names like on `manager.js` using `setup`.
I feel like there is no need for a `Tooltips`. We should move each piece of "Script" into it's own file, so it can be removed without issues.